test/fixtures/output/python/python3/application-json.py

```diff
@@ -0,0 +1,15 @@
+import http.client
+
+conn = http.client.HTTPConnection("mockbin.com")
+
+payload = "{\"number\": 1, \"string\": \"f\\\"oo\", \"arr\": [1, 2, 3], \"nested\": {\"a\": \"b\"}, \"arr_mix\": [1, \"a\", {\"arr_mix_nested\": {}}] }"
```

Comment:
ah, the syntax highlighting in github threw me off .. was checking this on mobile ...